import-service/handlers/catalogBatchProcess/catalogBatchProcess.ts

```diff
@@ -0,0 +1,23 @@
+import fetch from 'node-fetch';
+
+import apiSNS from 'services/apiSNS';
+
+export const catalogBatchProcess = async (event) => {
+  const { Records } = event;
+  const products = [];
+  Records.forEach(element => {
+    const { body } = element;
+    products.push(JSON.parse(body));
+  });
+
+  try {
+    await fetch('https://zrz0p4i0zk.execute-api.eu-west-1.amazonaws.com/dev/products', {
```

Comment:
Please look into the comment below/above (about moving handler to products service).
It would be better using direct db calls in product service instead of making fetch calls.

Reply:
So, i answer in the above comment. Anyway, I decide to do in such way and it is real matter how i implement it and what is your concerns?)